vyper/builtin_functions/functions.py

```diff
@@ -1545,12 +1546,15 @@ class CreateForwarderTo(_SimpleBuiltinFunction):
 
     _id = "create_forwarder_to"
     _inputs = [("target", AddressDefinition())]
-    _kwargs = {"value": Optional("uint256", zero_value)}
+    _kwargs = {"value": Optional("uint256", zero_value), "salt": Optional("bytes32", empty_value)}
     _return_type = AddressDefinition()
 
     @validate_inputs
     def build_LLL(self, expr, args, kwargs, context):
         value = kwargs["value"]
+        salt = kwargs["salt"]
+        is_deterministic = "salt" in [kwarg.arg for kwarg in expr.keywords]
```

Comment:
nit: i would change this variable name to `should_use_create2`

Reply:
As a great man once said:

> `should_*` for booleans that affect control flow

forgot the guy's name though :thinking: :laughing: